scripts/write_to_db.py

```diff
@@ -0,0 +1,47 @@
+import pandas as pd
+import numpy as np
+from spreadsheets_helper import read_spreadsheet
+from database_helper import connect_to_db, update_row_orm, close_connection
+
+def main():
+    SCOPES = ['https://www.googleapis.com/auth/spreadsheets']
+    SPREADSHEET_ID_INPUT = '1RZDZTswZxI-smwnyi8Oh5ck092hgl_CBxjtQURrczBs'
+    SPREADSHEET_RANGE ='A1:Z10'
+    credentials_file_name = 'credentials.json'
+    DATABASE_URL = ''
+
+    df = read_spreadsheet(SPREADSHEET_ID_INPUT, SPREADSHEET_RANGE, SCOPES)
+    df['id'] = df['id'].astype(int)
+    db_session = connect_to_db(DATABASE_URL)
+
+    for index, row in df.iterrows():
+        store_name = row['name']
+        flagged = row['flag']
+        deleted = row['deleted']
+        reviewed = row['Reviewed']
```

Comment:
just want to check `R` capital is intended? others are lowercase. it would be a good idea to use these as variables

Reply:
would be nice to be able to use the class attributes of `Store`

Reply:
Reviewed is the name of the column in G Sheets. It doesn't matter a huge deal whether it's capitalised or not, as long as it's reading it from G Sheets, so I didn't bother changing it. We do use the attributes of `Store` when updating the object. Note that we cannot use a `Store` object at this point because we have not fetched it (from the DB) yet.